client/src/app/users/user-card.component.ts

```diff
@@ -8,8 +8,8 @@ import { User } from './user';
 })
 export class UserCardComponent implements OnInit {
 
-  @Input() user: User;
-  @Input() simple ? = false;
+  @Input() user?: User;
```

Comment:
Why make this optional?

Reply:
There can't really be an initial value for it here since it's value is coming from somewhere else (hence the `@Input`). The Internet seems to suggest that `?` is the way to go here, but I confess to not being thrilled about this decision. Do you have another suggestion?